.azure-pipelines/callGithubWorkflow.yaml

```diff
@@ -60,7 +62,40 @@ jobs:
         # NOT MAIN/MASTER OR AUTOMATED BRANCH
         if ($branchName -ne 'master' -and $isAutoGeneratedPR -eq $false -and $targetBranch -eq 'master')
         {
-          .script/package-automation/invokeGithubWorklow.ps1 $branchName $pullRequestNumber
+          # INVOKE GITHUB WORKFLOW
+          $header = @{
+              "Accept" = "application/vnd.github+json"
```

Comment:
what is this header?

Reply:
we are making api call to github to run workflow for us.